webapp/src/components/cardDetail/__snapshots__/cardDetailContents.test.tsx.snap

```diff
@@ -0,0 +1,979 @@
+                    class="CompassIcon icon-dots-horizontal OptionsIcon"
+                  />
+                </div>
+              </div>
+              <div
+                class="dnd-handle"
+                draggable="true"
+              >
+                <svg
+                  class="GripIcon Icon"
+                  viewBox="0 0 24 24"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    d="M0 0h24v24H0V0z"
+                    fill="none"
+                  />
+                  <path
+                    d="M11 18c0 1.1-.9 2-2 2s-2-.9-2-2 .9-2 2-2 2 .9 2 2zm-2-8c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0-6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm6 4c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
+                  />
+                </svg>
+              </div>
+            </div>
+            <div
+              class="addToRow "
+              style="flex: 0 0 auto; height: 100%;"
+            />
+            <div
+              class="DividerElement"
+            />
+          </div>
+          <div
+            class="addToRow "
+          />
+        </div>
+        <div
+          class="rowContents"
+          style="width: 50%;"
+        >
+          <div
+            class="ContentBlock octo-block"
+          >
+            <div
+              class="octo-block-margin"
+            >
+              <div
+                class="MenuWrapper"
+              >
+                <div
+                  class="Button IconButton"
+                >
+                  <i
+                    class="CompassIcon icon-dots-horizontal OptionsIcon"
+                  />
+                </div>
+              </div>
+              <div
+                class="dnd-handle"
+                draggable="true"
+              >
+                <svg
+                  class="GripIcon Icon"
+                  viewBox="0 0 24 24"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    d="M0 0h24v24H0V0z"
+                    fill="none"
+                  />
+                  <path
+                    d="M11 18c0 1.1-.9 2-2 2s-2-.9-2-2 .9-2 2-2 2 .9 2 2zm-2-8c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0-6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm6 4c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z"
+                  />
+                </svg>
+              </div>
+            </div>
+            <div
+              class="DividerElement"
+            />
+          </div>
+          <div
+            class="addToRow "
+          />
+        </div>
+      </div>
+      <div
+        class="addToRow "
+        style="width: 94%; height: 10px; margin-left: 48px;"
+      />
+    </div>
+  </div>
+</div>
+`;
```

Comment:
@sbishel @hahmadia does anyone really review these 1k of lines inside the snapshot?

Reply:
I usually do a quick scan for keywords in each snapshot test to ensure what I am looking for is there but I don't look through ever line just keyword classes / text I'd expect to be present in the snapshot.

Reply:
> quick scan for keywords

Sounds like this https://testing-library.com/docs/queries/bytext which can probably be done by the test itself.

Reply:
Thanks Kamre